frontend/apps/reader/modules/readerlink.lua

```diff
@@ -563,6 +713,29 @@ function ReaderLink:onGoToPageLink(ges, use_page_first_link)
         --         ["start_y"] = 1201
         --         ["a_xpointer"] = "/body/DocFragment/body/div/p[12]/sup[3]/a[3].0",
         --     },
+        -- and when segments requested (example for a multi-lines link):
+        --     [3] = {
+        --         ["section"] = "#_doc_fragment_0_ Développement_de_l�économie_marchande",
```

Comment:
Is the weird encoding on développement and économie supposed to be there, just a GH display artifact, or …? :-)

I just find it distracting, sorry. :-D

Reply:
I'll fix that, it's a mix of latin1 and utf8 cut & pasted from koreader debug log output in my latin1-only terminal , and GH doing what it can with that :)